src/utils/policies/utils.ts

```diff
@@ -11,3 +13,19 @@ export const getPolicyEnactments = (
           enactment?.metadata?.labels?.[ENACTMENT_LABEL_POLICY] === policy?.metadata?.name,
       )
     : [];
+
+export const isPolicyAppliedInNode = (
+  policy: V1NodeNetworkConfigurationPolicy,
+  node: IoK8sApiCoreV1Node,
+) => {
+  if (isEmpty(policy.spec.nodeSelector)) return true;
```

Comment:
Do we need optional chaining operators here?

Reply:
yes